app/js/lib/foxbox/settings.js

```diff
@@ -89,8 +87,10 @@ export default class Settings extends Model {
 
   set localOrigin(origin) {
     this._localOrigin = origin ?
-        (new URL(origin)).origin : DEFAULT_LOCAL_ORIGIN;
-    storage.setItem(`${PREFIX}localOrigin`, this._localOrigin);
+        (new URL(origin)).origin : undefined;
+    if (this._localOrigin) {
+      storage.setItem(`${PREFIX}localOrigin`, this._localOrigin);
+    }
```

Comment:
I thought that @gmarty was asking to call ```removeItem``` on the localStorage in case ```!this._localOrigin``` to keep these values in sync, but I'm not sure too :)